src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml.cs

```diff
@@ -49,7 +49,7 @@ public ShortcutToolbar(DynamoViewModel dynamoViewModel)
 
             InitializeComponent();         
 
-            var shortcutToolbar = new ShortcutToolbarViewModel(dynamoViewModel);
+            var shortcutToolbar = new ShortcutToolbarViewModel(this, dynamoViewModel);
```

Comment:
I think this would cause memory leak.. usually we do not keep a reference of the view in view model. because view model will not be able to dispose before view. You may want to again use event. e.g. have the view watch for view model property changed

Reply:
Ok, so I moved the code to the code behind itself. Please review now. Thanks